bot/config/__init__.py

```diff
@@ -63,3 +64,7 @@ class TCGConfig:
         TCG_TITLE_ROLE_ID[2]: '<:MasterDuelist:1052440400822018078>',
         TCG_TITLE_ROLE_ID[3]: '<:ImmortalDuelist:1052440404135518228>',
     }
+
+
+class Booster:
+    BOOSTER_ROLE_ID = 768874803712753727
```

Comment:
It's not a good practice to store only a single attribute within a class. So let's move it out to the other role id constants.